src/components/simple-color-picker/simple-color-picker.component.tsx

```diff
@@ -260,9 +260,15 @@ export const SimpleColorPicker = React.forwardRef<
   const handleOnBlur = (ev: React.FocusEvent<HTMLInputElement>) => {
     ev.preventDefault();
 
-    if (!blurBlocked && onBlur) {
-      onBlur(ev);
-    }
+    setTimeout(() => {
+      const hasBlurred = !gridItemRefs?.current?.find(
+        (colorRef) => colorRef === document.activeElement
+      );
+      /* istanbul ignore else */
+      if (onBlur && hasBlurred && !blurBlocked) {
```

Comment:
comment (non-blocking): I'm not convinced we need the `blurBlocked` flag anymore as the new find function should handle it as far as I can tell. If that's the case, maybe we should remove the internal state and add a deprecation warning for it? It doesn't seem to cause any issues leaving it in for now (I do think getting the dep warning in would be good) so happy to push this to a new ticket if preferred.

Reply:
Yeah that is a good point. I did originally get rid of all the `blurBlocked` stuff (state/prop) and it worked with the amended blur handling code. I chose to add it back in otherwise it would have been a breaking change. I'll gladly put a deprecation warning on the prop and we can add it to the next bundle of breaking changes we do.